src/Middleware/PermissionMiddleware.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Spatie\Permission\Middleware;
+
+use Auth;
+use Closure;
+
+class PermissionMiddleware
+{
+    public function handle($request, Closure $next, $permission)
+    {
+        if (Auth::guest()) {
+            return $next($request);
```

Comment:
The simplest solution would be to just `abort(403)`. People can do the redirection in the exception handler if they wish.